lua/handler_group.lua

```diff
@@ -0,0 +1,48 @@
+local SS13 = require('SS13')
+local HandlerGroup = {}
+
+HandlerGroup.metatable = {}
+HandlerGroup.metatable.__index = HandlerGroup.metatable
```

Comment:
You don't need or really want this `.metatable` thing. The most idiomatic way to do this in Luau is what's been nicknamed "one true class":

```lua
local HandlerGroup = {}
HandlerGroup.__index = HandlerGroup

function HandlerGroup.new()
    return setmetatable({
        -- props
    }, HandlerGroup)
end
```